src/commands/synthetics/utils/internal.ts

```diff
@@ -87,3 +88,97 @@ export const toExecutionRule = (env: string | undefined): ExecutionRule | undefi
 
   return undefined
 }
+
+type validTestOverridesValues = 'boolean' | 'number' | 'string' | 'string[]' | 'ExecutionRule'
+type AccumulatorBaseConfigOverride = Omit<
+  BaseConfigOverride,
```

Comment:
Don't we want to use `UserConfigOverride` instead, to include `mobileApplicationVersion` and `mobileApplicationVersionFilePath`?